src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.Body.liquid

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Keep this file completely empty, or is it already?

Reply:
just checked.. it is empty.